renderdoc/driver/d3d12/d3d12_common.cpp

```diff
@@ -1120,13 +1120,13 @@ D3D12_SAMPLER_DESC2 ConvertStaticSampler(const D3D12_STATIC_SAMPLER_DESC1 &samp)
       desc.UintBorderColor[0] = desc.UintBorderColor[1] = desc.UintBorderColor[2] = 0;
       desc.UintBorderColor[3] = 1;
       // this flag is optional in D3D, add it here to ensure we can check it elsewhere
-      desc.Flags |= D3D12_SAMPLER_FLAG_UINT_BORDER_COLOR;
```

Comment:
Now the flags are initialized please can we keep the "|=" lines : it is a common pattern to use "|="

When making this change please edit the commit history to ensure there is a single commit and not two commits

Reply:
I believe I have succeeded in doing that. So we're just looking at a single line change. Perfect.